ibm_catalog.json

```diff
@@ -0,0 +1,219 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-secrets-manager-private-cert",
+        "label": "Cloud automation for Secrets Manager private certificate",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "security"
+        ],
+        "keywords": [
+          "Private certificate",
+          "Secrets Manager",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution"
+        ],
+        "short_description": "Creates and configures a Secrets Manager private certificate.",
+        "long_description" : "This deployable architecture is used to create a private certificate within an existing instance of IBM Cloud Secrets Manager. It requires that the Secrets Manager instance is already provisioned and configured with a Private Certificate Engine, which is necessary for managing and issuing private certificates. The module simplifies the process of provisioning certificates by automating the necessary infrastructure steps, ensuring secure and consistent certificate creation aligned with your organization's security policies.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-private-cert/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-secrets-manager-private-cert/main/images/secrets_manager_private_cert.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Creates a private certificate template.",
+            "description": "Creates and manages a private certificate in an existing Secrets Manager instance that has a configured Private Certificate Engine."
+          }
+
+        ],
+        "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [here](https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-private-cert/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+        "flavors": [
+          {
+            "label": "Fully configurable",
+            "name": "fully-configurable",
+            "install_type": "fullstack",
+            "working_directory": "solutions/fully-configurable",
+            "architecture": {
+              "descriptions": "This architecture supports creating and configuring a Secrets Manager Private Certificate.",
```

Comment:
We dont need this description. Not sure which template you followed. It has been removed from all the existing  DAs.